### ISSUE TITLE
[GENERAL] What's the point of DmgModifier in creature_template?

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/forum#76]
```sql
SELECT entry,name,minlevel,subname, DamageModifier
FROM `creature_template`
WHERE `DamageModifier` > '1'
```
Result: 8,178 rows

8k rows use this column to change the damage of the mobs, but why? why not use min_dmg, and max_dmg?

What's the point besides making things more unmanageable? I don't understand at all. The only thing I can think of is that it helps people modify the creatures for a custom server, but even then it could be done with a simple update query.

**Comment by gengarshadowball** [comment]
I agree on this, it's a feature that is rarely used if ever. However would have to check the cases where it would hurt removing and update them with x times the value from the modifier on the damage.
I doubt it is wise to remove it because there might be servers using this 😕

**Comment by BarbzYHOOL** [comment]
sure but it's just confusing no? i'd like to know if there is a real NEED for this

**Comment by gengarshadowball** [comment]
Things like mod-autobalance might find a good use to this column, maybe...
